### PR TITLE
Change prefix

### PR DESCRIPTION
**Pull request by OpenSteveMap**
prefix changed to: "[YAGA](https://yagajs.org) | [leaflet-ng2](https://leaflet-ng2.yagajs.org)"

### CODE REVIEW

File: ts/attribution-control.directive.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class AttributionControlDirective extends Control.Attribution implements
     constructor(
         @Inject(forwardRef(() => MapComponent)) mapComponent: MapComponent
     ) {
-        super({prefix: '<a href="http://yagajs.org" title="A TypeScript library for interactive maps with Leaflet in Angular2">YAGA</a>'});
+        super({prefix: '<a href="https://yagajs.org" title="YAGA">YAGA</a> | <a href="https://leaflet-ng2.yagajs.org" title="A TypeScript library for interactive maps with Leaflet in Angular2">leaflet-ng2</a>'});
```
Comment (atd-schubert) [inline ts/attribution-control.directive.ts]: This line exceeded the max-line-length of 140 chars. There are two ways:

1. Put the string content into a constant and place it on the top of the file or in consts.ts
2. Add an exception for linting inline before.

#### Example for Number 2:
```ts
/* tslint:disable:max-line-length */
super({prefix: '...'});
/* tslint:enable */
```

**Review by atd-schubert — CHANGES_REQUESTED** [review]
Lint and build failed, this should be fixed first!

File: ts/attribution-control.directive.ts (hunk 1 above)
Comment (atd-schubert) [reply ts/attribution-control.directive.ts]: Even after the last commit this should stell be fixed!

Comment (OpenSteveMap) [reply ts/attribution-control.directive.ts]: i've broke the line into two lines

Comment (OpenSteveMap) [reply ts/attribution-control.directive.ts]:
```
super({prefix: '<a href="https://yagajs.org" title="YAGA">' + 
'YAGA</a> | <a href="https://leaflet-ng2.yagajs.org" title="Leaflet in Angular2">leaflet-ng2</a>'});
```

**Comment by atd-schubert** [comment]
Additionally I found an error in `removeAllAttributions`. The loop can not work because `i` get never a value in this case `0`. I will make these changes and make a commit...

**Comment by atd-schubert** [comment]
What do you think about this @OpenSteveMap ?

**Review by atd-schubert — APPROVED** [review]

**Pull request merged**